content/en/hosting/analytics/building-dbt-models.md

```diff
@@ -37,12 +37,18 @@ packages:
 ```
 To avoid breaking changes in downstream models, include `revision` in the dependency, which should be a version tag for `cht-pipeline`.
 
-In the CHT Sync config, set the URL of dbt GitHub repository to the `CHT_PIPELINE_BRANCH_URL` [environment variable]({{< relref "hosting/analytics/environment-variables" >}}), either in `.env` if using Docker compose, or in `values.yaml` if using Kubernetes.
+For production and remote deployments, in the CHT Sync config, set the URL of the dbt GitHub repository to the `CHT_PIPELINE_BRANCH_URL` [environment variable]({{< relref "hosting/analytics/environment-variables" >}}), either in `.env` if using Docker compose, or in `values.yaml` if using Kubernetes.
 
 {{% alert title="Note" %}}
 If `CHT_PIPELINE_BRANCH_URL` is pointing to a private GitHub repository, you'll need an access token in the URL. Assuming your repository is `medic/cht-pipeline`, you would replace  `<PAT>`  with an access token: `https://<PAT>@github.com/medic/cht-pipeline.git#main`. Please see [GitHub's instructions](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens) on how to generate a token. If you create a fine-grained access token you need to provide read and write access to the [contents](https://docs.github.com/en/rest/authentication/permissions-required-for-fine-grained-personal-access-tokens?apiVersion=2022-11-28#repository-permissions-for-contents) of the repository.
 {{% /alert %}}
 
+### Local development
+
+It is helpful to test changes locally before commiting them to a remote repository.
+For local testing and development, in the CHT Sync config, set the path to the dbt project to the `DBT_LOCAL_PATH` [environment variable]({{< relref "hosting/analytics/environment-variables" >}}) in `.env`
+Use the [docker compose setup]({{< relref "hosting/analytics/setup-docker-compose" >}}) for CHT Sync with the `--local` profile.
```

Comment:
I think we can be a bit more explicit about why using a local directory is better/faster

```suggestion
When doing development, it  is convenient to be able to test changes locally with out needing to push to a remote repository and then have dbt pull those same changes back down. To expedite local development, set the `DBT_LOCAL_PATH` [environment variable]({{< relref "hosting/analytics/environment-variables" >}}) to point to the dbt models directory on your computer.  Then, use the the `--local` profile [option]({{< relref "hosting/analytics/setup-docker-compose" >}}) when calling `docker` which avoids the `CHT_PIPELINE_BRANCH_URL` and uses the  `DBT_LOCAL_PATH` you just declared.
```